test/e2e/kubectl/kubectl.go

```diff
@@ -1074,15 +1074,6 @@ metadata:
 
 			meta := fmt.Sprintf(metaPattern, crd.Crd.Spec.Names.Kind, crd.Crd.Spec.Group, crd.Crd.Spec.Versions[0].Name, "test-cr")
 
-			// XPreserveUnknownFields is defined on the root of the schema so unknown fields within the spec
```

Comment:
had to drop this testcase which only works with server-side field validation (was added when promoting to beta)